src/components/Button/index.js

```diff
@@ -1,52 +1,87 @@
 import React from 'react';
 import cn from 'classnames';
+
 export const BUTTON_OPTIONS = {
   PADDING: {
-    ZERO: false, // no padding at all
-    DEFAULT: 'default', // default padding == 2rem
     SMALL: 'small', // for footer section
-    BIG: 'big', // big padding == 8rem,
-    MOB_PADDING: 'mobile_padding'
+    BIG: 'big' // big padding == 8rem,
   },
-  WIDTH: {
-    SMALL: 'small', // 8rem
-    BIG: 'big' //16rem
+  backgroundColor: {
+    C100: 'blue',
+    C200: 'red',
+    C300: 'black1',
+    C400: 'black2',
+    C500: 'grey1',
+    C600: 'grey2',
+    C700: 'grey3',
+    C800: 'grey4',
+    C900: 'off white',
+    C1000: 'green',
+    WHITE: 'white'
   },
-  HIEGHT: {
-    SMALL: 'small', // 2.5rem
-    BIG: 'big' // 3rem
+  COLOR: {
+    C100: 'blue',
+    C200: 'red',
+    C300: 'black1',
+    C400: 'black2',
+    C500: 'grey1',
+    C600: 'grey2',
+    C700: 'grey3',
+    C800: 'grey4',
+    C900: 'off white',
+    C1000: 'green',
+    WHITE: 'white'
```

Comment:
Avoid using color names that imitate/leak configuration information to the user/client. 
Try using names that give semantic meaning to the use of the color in the context of the web page, like: primary1, primary2, secondary1, secondary2, tertiary1, tertiary2, ... etc/

Reply:
I like the comment with one tweak. 

instead of ~`primary1`~ and ~`primary2`~ you should be more explicit, like `primaryLight` or `primaryDrak`, `secondaryCatchy`, `secondarydimmed`.. and so on